ui/src/views/Quickstart/index.jsx

```diff
@@ -192,6 +192,31 @@ const cmdDirectory = (type, org = '<YOUR_ORG>', repo = '<YOUR_REPO>') =>
     marginLeft: theme.spacing.unit,
     marginTop: theme.spacing.double,
   },
+  taskShouldRunFlex: {
+    display: 'flex',
+    width: '100%',
+    justifyContent: 'space-between',
+    [theme.breakpoints.down('xs')]: {
+      flexDirection: 'column',
+    },
+  },
+  formLabelSpacing: {
+    marginTop: -0.5 * theme.spacing.unit,
+    marginBottom: -0.5 * theme.spacing.unit,
+    marginRight: 4 * theme.spacing.triple,
+    [theme.breakpoints.down('sm')]: {
+      marginRight: theme.spacing.double,
+    },
+  },
+  mainHeading: {
+    marginTop: -theme.spacing.triple,
+  },
+  generalSpacing: {
+    marginTop: -2 * theme.spacing.unit,
+  },
+  descriptionSpacing: {
+    marginBottom: 2 * theme.spacing.unit,
+  },
 }))
```

Comment:
![Screen Shot 2019-10-17 at 3 10 56 PM](https://user-images.githubusercontent.com/3766511/67039901-562b3400-f0f0-11e9-8b05-a7ef87f9e2fd.png)

The organization text field is not properly aligned with "Name". Let's remove the left and right paddings. Same thing with the right side, align the repository text field with the name text field.